airbyte-integrations/connectors/source-zoho-crm/source_zoho_crm/streams.py

```diff
@@ -37,14 +37,21 @@ def next_page_token(self, response: requests.Response) -> Optional[Mapping[str,
     def request_params(
         self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, any] = None, next_page_token: Mapping[str, Any] = None
     ) -> MutableMapping[str, Any]:
-        return next_page_token or {}
+        params = super().request_params(stream_state=stream_state, stream_slice=stream_slice, next_page_token=next_page_token)
+        if self.module:
+            if len(self.module.fields) > 50:
+                self.logger.warning("ZohoCRM only allows a max of 50 parameters per query, dropping all fields after 50")
+            params["fields"] = ",".join([field.api_name for field in self.module.fields][:50])
```

Comment:
need describe this logic in docs file below

Reply:
Hey @lazebnyi. Sorry for the delay, where exactly do I need to describe the logic in the docs? In the Notes section, the changelog description or some other place?

Reply:
@lazebnyi I've added a comment in the streams.py file as well as described why fields after 50 are dropped in the notes section of the docs. Do let me know if there are any further changes.